crates/core/src/db/datastore/system_tables.rs

```diff
@@ -1095,7 +1103,7 @@ mod tests {
         let db = TestDB::durable().expect("failed to create db");
         let ctx = ExecutionContext::default();
         let _ = db.with_auto_commit(&ctx, |tx| {
-            StVarTable::write_var(&ctx, &db, tx, StVarName::RowLimit, StVarValue::U64(5))
+            StVarTable::write_var(&ctx, &db, tx, StVarName::RowLimit, "5")
```

Comment:
It should not be an `AlgebraicValue` that matches the `typeof` of `variables`?

Reply:
This is mainly for UX. I figured it's nicer to write `SET row_limit TO 5` rather than `SET row_limit TO (u64 = 5)`